### PR TITLE
webots.min.js/macOS: Fix `Ctrl + left click`

### PR DESCRIPTION
**Pull request by fabienrohrer**
Fix https://github.com/omichel/robotbenchmark/issues/303

- [x] On macOS, deal with "Ctrl + left click" as a right click
- [x] Use SystemInfo
- [x] Log

### CODE REVIEW

**Comment by fabienrohrer** [comment]
👍

File: resources/web/wwi/editor.js [hunk 1]
```diff
@@ -37,10 +37,10 @@ class Editor extends DialogWindow { // eslint-disable-line no-unused-vars
     this.menu = document.createElement('div');
     this.menu.id = 'webotsEditorMenu';
     var saveShortcut;
-    if (navigator.appVersion.indexOf('Mac') === -1)
-      saveShortcut = 'Ctrl-S';
-    else // macOS
+    if (SystemInfo.isMacOS())
       saveShortcut = 'Cmd-S';
+    else // macOS
```
Comment (stefaniapedrazzi) [inline resources/web/wwi/editor.js]:
```suggestion
    else
```

**Comment by stefaniapedrazzi** [comment]
:eyes:

**Comment by fabienrohrer** [comment]
👍

**Pull request merged**